packages/models-library/src/models_library/api_schemas_webserver/users_preferences.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, TypeAlias
+
+from pydantic import BaseModel, Field
+
+from ..user_preferences import PreferenceIdentifier
+
+
+class FrontendUserPreference(BaseModel):
```

Comment:
If this is the response payload, please use ``from ._base import OutputSchema`` as base. 
Then please remove all `Config` and `alias`es. Same with body, please inherit from `InputSchema`